src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+
```

Comment:
it's a square, so u can make a `@mixin` and add it here by `@include`.
that's not critical, but good for sass training)